code/modules/SCP/SCPs/SCP-049.dm

```diff
@@ -37,18 +33,21 @@ GLOBAL_LIST_EMPTY(scp049_1s)
 	return
 
 /mob/living/carbon/human/scp049/Initialize()
-	..()
-	add_language(LANGUAGE_ENGLISH, TRUE)
```

Comment:
TRUE meant that 049 could speak that language.
I'm not sure if it's set to TRUE by default on the proc itself, so it might be a good idea to test it.

Reply:
I wasn't quite sure if it actually did anything. When I checked the proc it seemed like it only accepted the language. I'll check again.